src/nlp-cvx/109_010.jl

```diff
@@ -1,25 +1,26 @@
-function nlp_cvx_109_010(optimizer, objective_tol, primal_tol, dual_tol)
+function nlp_cvx_109_010(optimizer, objective_tol, primal_tol, dual_tol;
+        termination_target = JuMP.MOI.LOCALLY_SOLVED,
+        primal_target = JuMP.MOI.FEASIBLE_POINT)
     # Test Goals:
     # - convex logarithmic objective
     # - binding nonlinear constraint
     # Variants
     #   010 - binding constraint (inflection point)
     #   011 - binding constraint (non-inflection point)
     #   012 - binding constraint (non-inflection point)
-    
-    m = Model(solver = optimizer)
-    
-    @variable(m, x, start=0.1)
-    @variable(m, y, start=0.1)
-    
+
+    m = Model(optimizer)
+
+    @variable(m, x, start=2)
+    @variable(m, y, start=2)
```

Comment:
@ccoffrin I had to change the starting points because Ipopt kept running into numerical error. Is that a problem?

Reply:
In my opinion, we should not set a initial point being equal to the optimal one. I think it is better to understand why Ipopt does not converge when `start=0.1` rather than modifying the continuous integration test...

Reply:
My bad, sorry I missed this before.  I am in favor of restoring this starting point to the original `start=0.1`, at least until we have a better understanding of why Ipopt and Ipopt via NL are giving notably different results.  This example is sufficiently small that the root cause should be identifiable.  

I also think it is good that NL writer test omissions will highlight there is a difference between native Ipopt and Ipopt via NL.